src/bootstrap.tsx

```diff
@@ -37,8 +37,8 @@ const App = () => {
   useInitializeAnalytics();
 
   useEffect(() => {
-    const title = typeof documentTitle === 'string' ? `${documentTitle} | ` : '';
-    document.title = `${title}console.redhat.com`;
+    const title = typeof documentTitle === 'string' ? `${documentTitle} | Hybrid Cloud Console` : 'Hybrid Cloud Console';
```

Comment:
We have to ensure we are not trying to assign a different type to the title. We can use `useMemo` instead of `useEffect` to properly set the titles.

The difference between these two is that `useMemo` runs even on a first render in addition to the dependency changes. `useEffect` runs only after the first render (and dependency changes). 

This changes might remvoe some of the issues as well in addtion to the default changes.